### ISSUE TITLE
description in sinkCredential

### ISSUE THREAD

**Issue by jlurien** [camaraproject/Commonalities#271]
**Problem description**

The current syntax with description under `allOf` was reported to cause problems with generators
```
        sinkCredential:
          allOf:
            - description: A sink credential provides authentication or authorization information necessary to enable delivery of events to a target.
            - $ref: "#/components/schemas/SinkCredential"
```

**Expected behavior**
In this case `description` could be moved to `SinkCredential` schema; as it is generic enough to be reused in the 2 $ref to this schema, and the `allOf` above can be removed.

**Comment by rartych** [comment]
We need to close RC.2 very soon, but for me the proposed correction is important for all subscription related APIs. 
As we have still open PR #267 with changes in subscription template, I added the proposed changes.
@jlurien please review it.

**Comment by jlurien** [comment]
The change LGTM, but @PedroDiez will review formally the other PR on behalf of TEF

**Comment by rartych** [comment]
Closing as resolved in #267
